examples/dyn_phy.py

```diff
@@ -0,0 +1,86 @@
+import sys
+
+sys.path.append("/usr/local/serialbox/python/")
```

Comment:
This is scary. What's the hold up to having proper PYTHONPATH setup?

Reply:
we do this in fv3core as well, I think just out of convenience at the time (though I see in the profiler you did do the PYTHONPATH route).We could indeed set the PYTHONPATH in the dockerfile and any daint script. We can probably do a wholesale shift for everything all at once -- right now this matches the convention used else where :D

Reply:
at least, when we don't need serialized data here, at least in this file, it will be removed

Reply:
When you're ingesting a package which cannot be pip installed, you have to be ready for some amount of spookiness.